src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -170,10 +182,13 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
             ImmutableArray<BoundExpression> boundNamedArguments = analyzedArguments.NamedArguments;
             constructorArguments.Free();
 
-            return new BoundAttribute(node, attributeConstructor, boundConstructorArguments, boundConstructorArgumentNamesOpt,
+            var attribute = new BoundAttribute(node, attributeConstructor, boundConstructorArguments, boundConstructorArgumentNamesOpt, argsToParamsOpt, expanded,
                 boundNamedArguments, resultKind, attributeType, hasErrors: resultKind != LookupResultKind.Viable);
-        }
 
+            NullableWalker.AnalyzeIfNeeded(Compilation, attribute, diagnostics);
```

Comment:
This is going to present problems with the public API.

Reply:
Not saying it's something you can or need to fix here, but this information will have to be available to the semantic model somehow.

---
In reply to: [264445223](https://github.com/dotnet/roslyn/pull/33816#discussion_r264445223) [](ancestors = 264445223)

Reply:
How so? It's unclear what you mean here.

Reply:
@jaredpar the output from this walk will need to be communicated to the semantic model in some manner. Currently, there's no real facility here.